com.syntevo.SmartGit.json

```diff
@@ -51,18 +51,6 @@
           "url": "https://github.com/git-lfs/git-lfs/releases/download/v2.8.0/git-lfs-linux-amd64-v2.8.0.tar.gz"
         }
       ]
-    }, {
-      "name": "openssh",
```

Comment:
```javascript
"name": "openssh",
"buildsystem": "simple",
"build-commands": [
  "install -D -m0755 -t /app/bin/ $(which ssh)"
]
```

Reply:
@x80486 why this is needed? Can't app use `/usr/bin/openssh`?

Reply:
I can't remember, but I think some time ago the `ssh` and `git` binaries were hard-coded to `/app/bin/` when recognized as Flatpak package. I would need to confirm, but I think it's _discovered_ in such a way.

Reply:
@x80486 heh, I wonder if it would work with just symlinks instead of copying binaries?

Reply:
I think that's better...let me see if it works :sunglasses: